projects/lf-documentation/src/index.html

```diff
@@ -6,6 +6,7 @@
   <title>LfDocumentation</title>
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="icon" type="image/x-icon" href="favicon.ico" />
+  <link href="https://lfxstatic.com/npm/@laserfiche/lf-ui-components@14/cdn/indigo-pink.css" rel="stylesheet"/>
```

Comment:
this is kind of a chicken and egg problem because we will want to use the 15 version, but we haven't released the 15 version yet. Okay to keep 14, just wanted to note it